api/python/provisioner/commands/check.py

```diff
@@ -609,6 +609,106 @@ def _cluster_status(*, args: str) -> CheckEntry:
 
         return res
 
+    @staticmethod
+    def _storage_lvms(*, args: str) -> CheckEntry:
+        """Storage lvms check."""
+        res: CheckEntry = CheckEntry(cfg.Checks.STORAGE_LVMS.value)
+        try:
+            from cortx.utils.kvstore.pillar import PillarStorage
+            from cortx.utils.validator.v_storage import StorageV
+        except ImportError:
+            res.set_fail(checked_target=cfg.ALL_MINIONS,
+                         comment="Package cortx-py-utils not installed")
+            return res
```

Comment:
newlines might help to make the code slightly better

Reply:
comment is addressed